runtime/src/topo/hwloc/topo-hwloc.c

```diff
@@ -756,6 +832,11 @@ int chpl_topo_bindCPU(int id) {
   return status;
 }
 
+chpl_bool chpl_topo_isOversubscribed(void) {
+  CHK_ERR(initialized);
+  _DBG_P("oversubscribed = %s\n", oversubscribed ? "True" : "False");
+  return oversubscribed;
```

Comment:
I think this new code is no longer respecting the `CHPL_RT_OVERSUBSCRIBED` env var

Reply:
Fixed.